application/common/tizen/signature_parser.cc

```diff
@@ -111,6 +111,34 @@ xmlNsPtr GetNamespace(const xmlNodePtr node, const char* expected_href) {
 
 namespace xwalk {
 namespace application {
+int hexToInt(char a) {
```

Comment:
Look the code again, please  put `HexToInt()` and `DecodeProcent()` under anonymous namespace.